README.md

```diff
@@ -0,0 +1,167 @@
+
+## Hello, World!
+
+Now, as tradition demands, we should set up a "Hello, World" server don't you think? This will help introduce you to the basic pattern that we will use throughout the lab in order to build our server. As you may have noticed already, there is a test written for this use case, and naturally it is failing at the moment:
+
+![](http://ezmiller.s3.amazonaws.com/public/flatiron-imgs/hello_world_test.png)
+
+Because tests serve as a spec for the code we need to write (this is why they are so helpful!), we can see from the title of the test what it is we need to do: "GET request to / respond with 'Hello, World!'. We need to write code such that our server can accept a request of method "GET" to the root path `/` and then respond with a "Hello, World!" This shouldn't be too hard!
+
+Let's see if you can get this test to work. Since we're just getting started here, we've placed some boilerplate code in the project to get you going in the right direction. Take a look at the `server.js` file. This will be the file that contains our server logic. Looking in this file you should see a snippet like this:
+
+```
+const router = new Router();
+
+router.get('/', (request, response) => {
+  // A good place to start!
+  response.end();
+});
+```
+
+So what do we have here? First of all, `const router = new Router()` instantiates our "router." You will encounter this term "router" frequently in the world of web applications. Essentially, a router is the piece of a web application that matches certain url requests to the write pieces of code, sometimes called "business logic", which then are responsible for returning the desired result. 
+
+> Note: In our code we are using a pre-written router module for node. You installed that package in the setup stage of this lab, and it is imported in the `server.js` file with the command: `const Router = require('router')`. There are a great many such router modules; we are using [this one](https://github.com/pillarjs/router), which has been extracted fromt he popular [Express.js](http://expressjs.com/) web server framework for Node. We will be learning about Express later in this unit!
+
+The code snippet above, then, sets up a "route" from a request to the server root path, i.e. `http://localhost:3000/` to the code that is contained within the function that is passed to the router's `get` method. There's a lot to talk about here actually. 
+
+First, notice that the method passed into `get` (written with the ES6 "fat arrow" syntax) takes two arguments: `request` and `response`. These are very important objects and you will be using them a lot when you work with Node. For now, what you should know is that the `request` object contains information as well as method relating to the "request" that has come in from the client. The `response` object, as you may have guessed, contains information and methods about how the server's response.
```

Comment:
"These are very important objects and you will be using them a lot when you work with Node." -> "These are very important objects and we will be using them a lot when we work with Node."

"methods about how the server's response." -> "methods about the server's response."